quests/Wall_Orbuculum.md

```diff
@@ -1,24 +1,37 @@
 # Wall Orbuculum. Game "Winter is coming"
 
-Hey, Application Engineer, we would like you to create game client for existing game server :)
+Hey, Application Engineer, we would like you to create game client for existing game server 😃
+
+## Task
 
-## Requirements
 Let’s say you have some hipster friends who only use Mac Books and iPhones and are very picky about native user experience.
 But they heard about this awesome [game](../README.md#game-rules) and want to try it out.
 
-- Fulfill specification - implement desktop or mobile application by [UI specification](#ui-specification)
-- Integrate to Server - implement Client by [Communication Channel specification](Talk_to_Zombies.md#communication-channel-specification)
+You will have to implement desktop or mobile application, which should fulfill [UI specification](#ui-specification).
+
+Application should communicate to server, see [Communication Channel specification](Talk_to_Zombies.md#communication-channel-specification).
+Server is running at `winter-is-coming.mysterium.network` on port `8765`.
+
+This task is *technology agnostic* (we love any readable code) - choose a technology which you know well and which seems appropriate for this task. 
+
+## Requirements
+
+Your task will be evaluated on these criteria:
+
+- Implementation correctness of functionality described in task description
 - Concentration - only solve the requirements and don't spent more than 2 workdays
-- Technology agnostic - choose a technology for this problem solving (we love any readable code)
 - Clean code - demonstrate your most readable code
 - Design patterns - demonstrate your design skills (SOLID, KISS, DRY etc.)
 - Testing - demonstrate testing skills by writing automated tests for your solution (creepy love for tests is totally fine)
 - We can easily run your application and try it out
-- Share code on VCS and send a link to jobs@mysterium.network
```

Comment:
I would not remove this, in case some random developer finds this challenge

Reply:
This is not removed - I've moved it to a separate section `Submitting`